### ISSUE TITLE
FSIApplications tests are failing

### ISSUE THREAD

**Issue by philbucher** [KratosMultiphysics/Kratos#3629]
Release, Ubuntu 16.04, 64 bit, GCC 5.4, without Cotire

Found this while testing for #3375

~~~
DEPRECATION: ************************************************************
DEPRECATION: USING OLD DEPRECATED CONSTRUCTOR OF MODELPART
DEPRECATION: THIS WILL BE REMOVED ON NOV 1 2018
DEPRECATION: the call to ModelPart(Structure ) 
DEPRECATION: should be substituted by current_model.CreateModelPart(Structure) 
DEPRECATION: ************************************************************
DEPRECATION: ************************************************************
DEPRECATION: USING OLD DEPRECATED CONSTRUCTOR OF MODELPART
DEPRECATION: THIS WILL BE REMOVED ON NOV 1 2018
DEPRECATION: the call to ModelPart(MainModelPart ) 
DEPRECATION: should be substituted by current_model.CreateModelPart(MainModelPart) 
DEPRECATION: ************************************************************
E
======================================================================
ERROR: testMokBenchmark (mok_benchmark_test.MokBenchmarkTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_aux/applications/FSIapplication/tests/mok_benchmark_test.py", line 66, in testMokBenchmark
    self.solver = solver_module.CreateSolver(self.structure_main_model_part, self.fluid_main_model_part, self.ProjectParameters)
TypeError: CreateSolver() takes 2 positional arguments but 3 were given

----------------------------------------------------------------------
~~~

and the following compiler warning:
~~~
In file included from /home/philippb/software/Kratos_aux/applications/FSIapplication/custom_python/add_convergence_accelerators_to_python.cpp:25:0:
/home/philippb/software/Kratos_aux/applications/FSIapplication/custom_utilities/mvqn_convergence_accelerator.hpp: In instantiation of ‘void Kratos::MVQNFullJacobianConvergenceAccelerator<TSpace>::UpdateSolution(const VectorType&, Kratos::MVQNFullJacobianConvergenceAccelerator<TSpace>::VectorType&) [with TSpace = Kratos::UblasSpace<double, boost::numeric::ublas::matrix<double>, boost::numeric::ublas::vector<double> >; Kratos::MVQNFullJacobianConvergenceAccelerator<TSpace>::VectorType = boost::numeric::ublas::vector<double>]’:
/home/philippb/software/Kratos_aux/applications/FSIapplication/custom_python/add_convergence_accelerators_to_python.cpp:67:81:   required from here
/home/philippb/software/Kratos_aux/applications/FSIapplication/custom_utilities/mvqn_convergence_accelerator.hpp:219:24: warning: unused variable ‘svd_its’ [-Wunused-variable]
             const auto svd_its = SVDUtils<double>::SingularValueDecomposition(aux2, u_svd, w_svd, v_svd, svd_type, svd_rel_tol);
~~~

**Comment by rubenzorrilla** [comment]
@philbucher thanks for reporting. Probably the Mok benchmark in there is not up-to-date... I will take the chance to fix the warning as well.

**Comment by philbucher** [comment]
there are still some warnings like this:
~~~
DEPRECATION: USING OLD DEPRECATED CONSTRUCTOR OF MODELPART
DEPRECATION: THIS WILL BE REMOVED ON NOV 1 2018
DEPRECATION: the call to ModelPart(Test ModelPart ) 
DEPRECATION: should be substituted by current_model.CreateModelPart(Test ModelPart) 
DEPRECATION: ************************************************************
~~~

Now the Mok-test runs, but I haven't checked the results visually (I am not sure if you implemented automatic checks)

**Comment by philbucher** [comment]
ping @rubenzorrilla could you check those please?
Or close if it has been fixed in the meantime

**Comment by rubenzorrilla** [comment]
Thanks for the reminder. I'm checking the warnings right now. Concerning the Mok benchmark results, the expected ones are obtained. The results are check using the wall tip x-displacement and pressure (expected values collected in mok_benchmark_results.json).

**Comment by rubenzorrilla** [comment]
Warnings fixed in #3776
